injected/src/features/ua-ch-brands.js

```diff
@@ -0,0 +1,120 @@
+import ContentFeature from '../content-feature';
+import { DDGReflect } from '../utils';
+
+export default class UaChBrands extends ContentFeature {
+    constructor(featureName, importConfig, args) {
+        super(featureName, importConfig, args);
+
+        this.originalBrands = null;
+    }
+
+    init() {
+        this.shimUserAgentDataBrands();
+    }
+
+    /**
+     * Override navigator.userAgentData.brands to match the Sec-CH-UA header
+     */
+    shimUserAgentDataBrands() {
+        try {
+            // @ts-expect-error - userAgentData not yet standard
+            if (!navigator.userAgentData || !navigator.userAgentData.brands) {
+                this.log.info('shimUserAgentDataBrands - navigator.userAgentData not available');
+                return;
+            }
+
+            // @ts-expect-error - userAgentData not yet standard
+            this.originalBrands = [...navigator.userAgentData.brands];
+            this.log.info(
+                'shimUserAgentDataBrands - captured original brands:',
+                this.originalBrands.map((b) => `"${b.brand}" v${b.version}`).join(', '),
+            );
+
+            const mutatedBrands = this.applyBrandMutationsToList(this.originalBrands);
+
+            if (mutatedBrands.length) {
+                this.log.info(
+                    'shimUserAgentDataBrands - about to apply override with:',
+                    mutatedBrands.map((b) => `"${b.brand}" v${b.version}`).join(', '),
+                );
+                this.applyBrandsOverride(mutatedBrands);
+                this.log.info('shimUserAgentDataBrands - override applied successfully');
+            }
+        } catch (error) {
+            this.log.error('Error in shimUserAgentDataBrands:', error);
+        }
+    }
+
+    /**
+     * Replace Microsoft Edge with DuckDuckGo in the brands list to match Sec-CH-UA header
+     * @param {Array<{brand: string, version: string}>} list
+     * @returns {Array<{brand: string, version: string}>} - Modified brands array
+     */
+    applyBrandMutationsToList(list) {
+        if (!Array.isArray(list) || !list.length) {
+            this.log.info('applyBrandMutationsToList - no brands to mutate');
+            return [];
+        }
+
+        const mutated = list.filter((b) => b.brand !== 'Microsoft Edge WebView2');
+        if (mutated.length < list.length) {
+            this.log.info('Removed "Microsoft Edge WebView2" brand');
+        }
+
+        const edgeIndex = mutated.findIndex((b) => b.brand === 'Microsoft Edge');
+        if (edgeIndex !== -1) {
```

Comment:
Shouldn't this be checking for WebView2?

Reply:
Native removes WebView2 and then replaces Edge, which is what I'm emulating, so this is overwriting Edge while using its version number.